ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/SynchronizerConfiguration.java

```diff
@@ -220,7 +220,7 @@ public int getWorldStateMaxRequestsWithoutProgress() {
     private Range<Long> blockPropagationRange = Range.closed(-10L, 30L);
     private long downloaderChangeTargetThresholdByHeight = 20L;
     private UInt256 downloaderChangeTargetThresholdByTd = UInt256.of(1_000_000_000L);
-    private int downloaderHeaderRequestSize = 10;
+    private int downloaderHeaderRequestSize = 200;
```

Comment:
Somewhat unrelated, but I see no reason not to bump up this value.  

This will mean we request larger chunks of headers when running `DetermineCommonAncestorTask`.  We'll also request a larger number of checkpoint headers when downloading.